manager/src/main/java/com/redhat/service/smartevents/manager/workers/resources/BridgeWorker.java

```diff
@@ -75,6 +79,9 @@ public Bridge createDependencies(Work work, Bridge bridge) {
         // We don't need to wait for the Bridge to be READY to create the Error Handler.
         createErrorHandlerProcessor(bridge);
 
+        // Create DNS record
+        dnsService.createDnsRecord(bridge.getId());
```

Comment:
This method adds up to 5 seconds to the execution time (in the worst case scenario).
As far as I remember this method is now invoked by Quartz and I don't know if this additional time can reach some external/end-to-end transaction timeout 🤔

Reply:
afaik quartz has no timeout on the jobs (it's not even possible to configure it)